template/src/ontology/Makefile.jinja2

```diff
@@ -1518,6 +1518,12 @@ validate-all-tsv: $(ALL_TSV_FILES)
 # Editors Utilities
 # ----------------------------------------
 
+# This is an experimental target people that want to use ODK extended prefix map
+# can use to pull the (currently inofficial) OBO extended prefix map into the workspace
+# Only works with odkfull variant
+tmp/obo.epm.json: /tools/obo.epm.json
```

Comment:
I’d prefer if users avoided to refer to `tmp/obo.epm.json`, just in case we’d want to change the location later (this PR makes me think that we could later want to generalise the idea of “ODK-provided resources”, and thus maybe have a dedicated `resources` directory somewhere).

I’d suggest having a `EXTENDED_PREFIX_MAP` Makefile variable pointing to `tmp/obo.epm.json`, and users being instructed to only use that variable instead `tmp/obo.epm.json` directly.

Reply:
Alright, I hope I did that correctly!

Reply:
Looks good!